lib/logstash/inputs/opensearch.rb

```diff
@@ -166,6 +166,8 @@ class LogStash::Inputs::OpenSearch < LogStash::Inputs::Base
   # SSL
   config :ssl, :validate => :boolean, :default => false
 
+  config :ssl_certificate_verification, :validate => :boolean, :default => true
```

Comment:
I don’t think we can introduce new variable with default as true. Are you sure it is not breaking backward compatibility?

Reply:
I'm still testing

Reply:
Yes, it is not breaking anything. By default, the ssl_certificate_verification value should be true and verify will be set to false when we set it explicitly

Reply:
if someone is migrating from elasticsearch to opensearch for user name and password without ca_file, how will the config looks like? ( user shouldn't be expected to add ssl_certificate_verification = false )

Reply:
Yes, it will still work the same even if user didn't explicitly set ssl_certificate_verification = false in their configuration.
The sample configuration is shown below:

```
input {
    opensearch {
        hosts       => "https://hostname:port"   
        user        => "admin"
        password    => "admin"
        index       => "logstash-logs-%{+YYYY.MM.dd}"
        query       => "{ "query": { "match_all": {}} }"
    }
}